### ISSUE TITLE
Ubuntu Package Manager dislikes file permissions

### ISSUE THREAD

**Issue by tawheeler** [dimensio/libpng16-deb#1]
Thank you for creating this package!
I figured I should let you know about this. Do with it what you will.

```
The installation of a package which violates the quality standards isn't allowed. This could cause serious problems on your computer. Please contact the person or organisation who provided this package file and include the details beneath.

Details:
Lintian check results for /home/user/Downloads/libpng16-deb-master/libpng16-16-1.6.17_amd64.deb:
E: libpng16-16: control-file-has-bad-permissions shlibs 0664 != 0644
E: libpng16-16: maintainer-address-missing dimens.io
```

**Comment by lovell** [comment]
Thanks for reporting.

I should be able to fix the `maintainer-address-missing` bit but the `control-file-has-bad-permissions` message is probably ignorable - see https://github.com/jordansissel/fpm/pull/648#issuecomment-36962145

**Comment by lovell** [comment]
The `maintainer-address-missing` error is fixed in 35a55b1.

The `control-file-has-bad-permissions` error will be fixed as of the next release of `fpm` - see https://github.com/jordansissel/fpm/pull/942

Thanks again for reporting.
